packages/replay-experimental/src/components/ReactPanel.tsx

```diff
@@ -0,0 +1,218 @@
+import { createSelector } from "@reduxjs/toolkit";
+import { ExecutionPoint, Location, TimeStampedPoint } from "@replayio/protocol";
+import classnames from "classnames";
+import { CSSProperties, ReactNode, Suspense, useContext, useMemo, useState } from "react";
+import AutoSizer from "react-virtualized-auto-sizer";
+import { FixedSizeList as List } from "react-window";
+
+import { selectLocation } from "devtools/client/debugger/src/actions/sources/select";
+import AccessibleImage from "devtools/client/debugger/src/components/shared/AccessibleImage";
+import { getExecutionPoint, getThreadContext } from "devtools/client/debugger/src/reducers/pause";
+import {
+  ReactUpdateScheduled,
+  reactRendersIntervalCache,
+} from "replay-experimental/src/suspense/reactInternalsCaches";
+import IndeterminateLoader from "replay-next/components/IndeterminateLoader";
+import { FocusContext } from "replay-next/src/contexts/FocusContext";
+import { ReplayClientContext } from "shared/client/ReplayClientContext";
+import { UIThunkAction } from "ui/actions";
+import { seek } from "ui/actions/timeline";
+import { JumpToCodeButton, JumpToCodeStatus } from "ui/components/shared/JumpToCodeButton";
+import MaterialIcon from "ui/components/shared/MaterialIcon";
+import { getSourceIdsByUrl } from "ui/reducers/sources";
+import { getCurrentTime } from "ui/reducers/timeline";
+import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
+
+import cardsListStyles from "ui/components/Comments/CommentCardsList.module.css";
+import styles from "ui/components/Events/Event.module.css";
```

Comment:
Seems pretty fragile for this component to import styles from other packages. We don't have TypeScript checking us there so it seems we're likely to "break" the React styles by changes made in these other CSS files.

Reply:
Yep. This was me trying to get something that looked passably decent, as quickly as possible (and an example of why I was saying we could start extracting some reusable components or styles).

Reply:
I think we should copy/fork the styles for now.